tests/plugins/templates/_assets/test.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+// Register a templates definition set named "default".
+CKEDITOR.addTemplates( 'test', {
+	// The name of sub folder which hold the shortcut preview images of the
+	// templates.
+	imagesPath: '/tests//plugins/templates/_assets/images/',
```

Comment:
Couldn't this path be simplified to `./images/`?

```suggestion
	imagesPath: './images/',
```

If not, then please remove the double slashes as they can cause an error:

```suggestion
	imagesPath: '/tests/plugins/templates/_assets/images/',
```

Reply:
Can't find the path with ./images ir ./html so i remove the double slash.